packages/experimental/RadioGroup/src/RadioGroup/useRadioGroup.ts

```diff
@@ -20,13 +20,15 @@ export const useRadioGroup = (props: RadioGroupProps): RadioGroupInfo => {
 
   const state: RadioGroupState = {
     value: data.selectedKey,
+    disabled: disabled || false,
```

Comment:
Also, can be `!!disabled`. No need to change :)

Reply:
Thanks for the tip!